simulator/src/test/java/com/hedera/block/simulator/BlockStreamSimulatorTest.java

```diff
@@ -92,10 +92,24 @@ void start_exitByBlockNull() throws InterruptedException, ParseException, IOExce
         assertTrue(blockStreamSimulator.isRunning());
     }
 
+    void start_usingConfigurationConstructor()
+            throws InterruptedException, ParseException, IOException {
+        blockStreamSimulator = new BlockStreamSimulatorApp(configuration);
```

Comment:
Perhaps use local variables rather than member variables to ensure deterministic behavior ^^